experimental/packages/opentelemetry-instrumentation-http/test/integrations/esm.test.mjs

```diff
@@ -37,183 +51,229 @@ provider.addSpanProcessor(new SimpleSpanProcessor(memoryExporter));
 const instrumentation = new HttpInstrumentation();
 instrumentation.setTracerProvider(provider);
 
-describe('HttpInstrumentation ESM Integration tests', () => {
-  let port;
-  let server;
+const httpImports = [
+  {
+    style: 'import http from "http"',
+    createServer: httpA.createServer,
+    request: httpA.request,
+    get: httpA.get,
+  },
+  {
+    style: 'import * as http from "http"',
+    createServer: httpB.createServer,
+    request: httpB.request,
+    get: httpB.get,
+  },
+  {
+    style: 'import {...} from "http"',
+    createServer: httpCreateServerC,
+    request: httpRequestC,
+    get: httpGetC,
+  },
+];
 
```

Comment:
Reviewer note: Be sure to use the "Hide whitespace" option for looking at this diff. There is an indentation change, so the diff is much smaller without the whitespace.